tests/test_react_note_editor.py

```diff
@@ -202,6 +202,18 @@ def venue(self, openreview_client):
                             'deletable': True
                         }
                     }
+                },
+                'datetime_picker_field': {
+                    'description': 'this is a datetime picker widget',
+                    'order': 1,
+                    'value': {
+                        'param': {
+                            'type': 'date',
+                            'optional': True,
+                            'deletable': True,
+                            'range': [0, 9999999999999]
+                        }
+                    }
```

Comment:
so we don't need to specify an input?

Reply:
based on the widget rendering logic discussion type=date or date[] will render date picker
if this is not accurate any more we can discuss again